[ONNX] Add support for shrink ops (#66969)

BowenBao opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #69550
* #69549
* #69548
* #69547
* #69546
* #69544
* #68493
* __->__ #68492
* #68491
* #68490
* #68489

* Initial commit

* Fix flake issue

* Add test tags

Differential Revision: [D32483827](https://our.internmc.facebook.com/intern/diff/D32483827)

BowenBao committed:
[ONNX] Add support for shrink ops (#66969)

* Initial commit

* Fix flake issue

* Add test tags

[ghstack-poisoned]

msaroufim commented:
@msaroufim has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D32483827).

BowenBao committed:
Update on "[ONNX] Add support for shrink ops (#66969)"


* Initial commit

* Fix flake issue

* Add test tags

Differential Revision: [D32483827](https://our.internmc.facebook.com/intern/diff/D32483827)

[ghstack-poisoned]

msaroufim commented:
@msaroufim has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D32483827).

BowenBao committed:
Update on "[ONNX] Add support for shrink ops (#66969)"


* Initial commit

* Fix flake issue

* Add test tags

Differential Revision: [D32483827](https://our.internmc.facebook.com/intern/diff/D32483827)

[ghstack-poisoned]

msaroufim commented:
@msaroufim has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D32483827).
